.github/workflows/cicd-main.yml

```diff
@@ -1919,6 +1919,26 @@ jobs:
         rm -rf ${NEMO_OUTPUT_PATH}
         rm -rf ${HF_OUTPUT_PATH}
 
+
+  L2_NeMo_2_Conversion_Test_Llama_Embedding:
+    needs: [pre-flight, cicd-test-container-build]
+    uses: ./.github/workflows/_test_template.yml
+    with:
+      RUNNER: self-hosted-azure
+      SCRIPT: |
+        HF_ORI_PATH=/home/TestData/llm/models/llama_31_toy
```

Comment:
This says `llama_31_toy` but then `NEMO_MODEL_CONFIG=Llama32EmbeddingConfig1B` follows. So there is a 3.1 vs 3.2 mismatch which is a bit confusing.

Reply:
Is this OK?

Reply:
Yes its fine. Its just a toy model and 3.1. and 3.2 is the same.